modules/machineset-creating-azure-ultra-disk.adoc

```diff
@@ -260,9 +307,10 @@ $ oc debug node/<node-name> -- chroot /host lsblk
 +
 In this command, `oc debug node/<node-name>` starts a debugging shell on the node `<node-name>` and passes a command with `--`. The passed command `chroot /host` provides access to the underlying host OS binaries, and `lsblk` shows the block devices that are attached to the host OS machine.
```

Comment:
Per @JoelSpeed this should be the same but wanted to make sure it looks right in the [updated doc render](https://52831--docspreview.netlify.app/openshift-enterprise/latest/machine_management/control_plane_machine_management/cpmso-using.html#machineset-creating-azure-ultra-disk_cpmso-using).